homeassistant/components/freebox/router.py

```diff
@@ -89,7 +93,39 @@ async def update_all(self, now: datetime | None = None) -> None:
     async def update_device_trackers(self) -> None:
         """Update Freebox devices."""
         new_device = False
-        fbx_devices: list[dict[str, Any]] = await self._api.lan.get_hosts_list()
+
+        fbx_devices: list[dict[str, Any]] = []
+
+        # Access to Host list not available in bridge mode, API return error_code 'nodev'
+        if self.supports_hosts:
```

Comment:
Use the no condition to avoid horizontal indent

```python
        if not self.supports_hosts:
            return
```

Reply:
No because if I do that `fbx_devices.append(` will not be executed

Reply:
Is it ok for you ?

Reply:
OK